tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "incremental": true,
-    "target": "ES2020",
-    "module": "ES2020",
+    "target": "ES2022",
```

Comment:
In the previous approach, I used an top level await which was required this to be 2022 or other options suggested.
but now since we are using a different approach, I reverted these to what they were. thanks!